eng/pipelines/templates/stages/partner-release.yml

```diff
@@ -0,0 +1,62 @@
+trigger: none
```

Comment:
nit: but we should consider putting this under `/eng/pipelines` directly given it is a full pipeline.